### PR TITLE
[RW-7265][risk=no] UX Fix For PD

### PR DESCRIPTION
**Pull request by YintongMa**
To fix UX issues mentioned in the following PD feedback

https://docs.google.com/document/d/1DELrUv_ZHs2o7FfaMWh0LNqJqI7HS-wdmq-Chb7oWX4/edit?resourcekey=0-LWOAmDh6mrHo8i187stQKw#heading=h.r8duy69bid57

### CODE REVIEW

File: ui/src/app/pages/analysis/runtime-panel.tsx [hunk 1]
```diff
@@ -304,21 +303,22 @@ export const ConfirmDeleteRuntimeWithPD = ({onCancel, onConfirm, computeType, pd
...
       <p style={{...styles.confirmWarningText, gridColumn: 1, gridRow: 3}}>
-        Deletes your application configuration and cloud compute profile, but detaches your
+        Deletes your analysis environment, but detaches your
         persistent disk and saves it for later.
         The disk will be automatically reattached the next time you create a cloud environment using
-        the standard VM compute type.
+        the standard VM compute type within this workspace.
       </p>
       <p style={{...styles.confirmWarningText, gridColumn: 1, gridRow: 4}}>
         You will continue to incur persistent disk cost at {formatUsd(diskPricePerMonth * pdSize)} per month.
```
Comment (calbach) [inline ui/src/app/pages/analysis/runtime-panel.tsx:320]: nit: please bold the price, including "per month" (Terra UI does this)

File: ui/src/app/pages/analysis/runtime-panel.tsx [hunk 2]
```diff
@@ -304,21 +303,22 @@ export const ConfirmDeleteRuntimeWithPD = ({onCancel, onConfirm, computeType, pd
...
+        You can delete your disk at any time  via the runtime panel.
```
Comment (calbach) [inline ui/src/app/pages/analysis/runtime-panel.tsx:321]: 2spaces -> 1space `time  via` ; sorry - that was a typo in my doc

File: ui/src/app/pages/analysis/runtime-panel.tsx [hunk 3]
```diff
@@ -328,11 +328,11 @@ export const ConfirmDeleteRuntimeWithPD = ({onCancel, onConfirm, computeType, pd
...
-          <label>Delete everything, including persistent disk</label>
+          <label>Delete persistent disk and environment</label>
         </div>
       </h3>
       <p style={{...styles.confirmWarningText, gridColumn: 1, gridRow: 2}}>
-        Deletes your persistent disk, which will also delete all files on the disk.
+        Deletes your persistent disk, which will also delete all files on the disk. Also deletes your analysis environment.
       </p>
       <p style={{...styles.confirmWarningText, gridColumn: 1, gridRow: 3}}>
         Also deletes your application configuration and cloud compute profile
```
Comment (calbach) [inline ui/src/app/pages/analysis/runtime-panel.tsx:338]: remove

**Review by calbach — COMMENTED** [review]
thanks!

**Review by calbach — APPROVED** [review]

**Review by yonghaoy — APPROVED** [review]

**Pull request merged**